scripts/updates/github-projects.txt

```diff
@@ -737,3 +737,5 @@ zstd|facebook/zstd
 
 ## Returns 'null' in latest tag.
 #zziplib|gdraheim/zziplib
+
+helix/helix-editor/helix
```

Comment:
Wrong format, first `/` should be a`|`.

Reply:
Also, put it in alphabetical order in the file.

Reply:
> first / should be a |

Sorry! That was a typo 😄